Core/Core/Models/Base.cs

```diff
@@ -278,7 +280,7 @@ public Base ShallowCopy()
       {
         myDuplicate[kvp.Key] = kvp.Value;
       }
-      catch (Exception)
+      catch (Exception ex) when (!ex.IsFatal())
```

Comment:
I guess there's no opportunity for logging anything here, eh?

Reply:
I'll add a warning log. But I have a feeling we can avoid this entirely / make the exception type more specific